src/controllers/playback/video/index.js

```diff
@@ -1227,50 +1233,22 @@ export default function (view) {
                     e.stopPropagation();
                 }
                 break;
-            case 'k':
-                playbackManager.playPause(currentPlayer);
-                showOsd(btnPlayPause);
-                break;
             case 'ArrowUp':
-            case 'Up':
```

Comment:
I don't think it should be split up.

Reply:
Ah that's my mistake. I just added a new commit to fix this and the other similar mistakes